tools/generate-helpers/pg-table-bindings/store_test.go.tpl

```diff
@@ -145,7 +145,7 @@ func (s *{{$namePrefix}}StoreSuite) TestStore() {
 	s.NoError(err)
 	s.Equal(200, {{.TrimmedType|lowerCamelCase}}Count)
 
-	s.NoError(store.DeleteMany(ctx, {{$name}}IDs))
+	s.NoError(store.PruneMany(ctx, {{$name}}IDs))
```

Comment:
Note: `DeleteMany` is not a generated function anymore, the code generator exposes the function of the Store object obtained from the `pkg/search/postgres` package. `DeleteMany` is already tested in `pkg/search/postgres`.

Reply:
I guess the same applies to `PruneMany`.  As such this change doesn't matter and removing it will greatly reduce the file count on the PR.  So I think I will undo this change.